modes/passenger-services.md

```diff
@@ -0,0 +1,204 @@
+# Mobility Data Specification: **Passenger Services**
+
+<img src="https://i.imgur.com/plW2Hon.png" width="120" align="right" alt="MDS Modes - Passenger Services" border="0">
+
+**Passenger Services** refers to taxis, transportation network companies (TNCs), commercial transport apps (CTAs), and private hire vehicles (PHVs).  Passenger Services typically have a driver, one or more passengers, and multiple passengers may be on different trips.  The state machine tracks the trip states of the passengers separately from the vehicle state.  
+
+See the [modes overview](/modes) for how the mode specific information below applies across MDS.
+
+## Taxi vs. TNC implementation differences
+
+Taxis typically require explicit tracking of maintenance while TNCs typically do not.
+
+## Table of Contents
+
+- [Mode Attributes](#mode-attributes)
+- [Vehicle States](#vehicle-states)
+- [Event Types](#event-types)
+- [Vehicle State Events](#vehicle-states-events)
+- [State Machine Diagram](#state-machine-diagram)
+
+## Mode Attributes
+
+### Mode ID
+
+The short name identifier for Passenger Services used across MDS is `passenger-services`.
+
+### Journey ID
```

Comment:
I think we should add some detail or examples here about how journeys and trips should connect. Like showing a journey with 3 pooled ride trips, a solo point to point trip, and awaiting_reservation or pickup_request trip. Not sure if we can describe all possible scenarios, or need to give examples with diagrams.